docs/book/src/migration/manually_migration_guide_v2_v3.md

```diff
@@ -0,0 +1,584 @@
+	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
+```
+
+With: 
+
+```go
+func main() {
+	var metricsAddr string
+	var enableLeaderElection bool
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
+```
+
+And then, rename the flags in the `config/default/manager_auth_proxy_patch.yaml` and `config/default/manager.yaml`:
+
+```yaml
+- name: manager
+args:
+- "--health-probe-bind-address=:8081"
+- "--metrics-bind-address=127.0.0.1:8080"
+- "--leader-elect"
+```
+
+#### Verification
+
+Finally, we can run `make` and `make docker-build` to ensure things are working
+fine.
+
+## Change your project to remove the Kubernetes deprecated API versions usage
+
+<aside class="note">
+<h1>Before continuing</h1>
+
+Make sure you understand [Versions in CustomResourceDefinitions][custom-resource-definition-versioning].
+
+</aside>
+
+
+The following steps
+will describe a workflow option to upgrade your project in order to remove the Kubernetes API versions usage which are deprecated `apiextensions.k8s.io/v1beta1`, `admissionregistration.k8s.io/v1beta1`, `cert-manager.io/v1alpha2`.
+
+Currently, the Kubebuilder CLI tool does not support scaffold resources for both Kubernetes API versions such as; an API/CRD with `apiextensions.k8s.io/v1beta1` and another one with `apiextensions.k8s.io/v1`.
+
+<aside class="note">
+<h1>Cert Manager API</h1>
+
+If you scaffold a webhook using the Kubernetes API `admissionregistration.k8s.io/v1` then, by default, it will use the API `cert-manager.io/v1` in the manifests. 
+
+</aside>
+
+The first step is to update your `PROJECT` file by replacing the `api.crdVersion:v1beta` and `webhooks.WebhookVersion:v1beta` with `api.crdVersion:v1` and `webhooks.WebhookVersion:v1` which would look like: 
+
+```yaml
+domain: my.domain
+layout: go.kubebuilder.io/v3
+projectName: example
+repo: example
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  group: webapp
+  kind: Guestbook
+  version: v1
+  webhooks:
+    defaulting: true
+    webhookVersion: v1
+version: "3"
+```
+
+Now, you can try to re-create the APIS(CRDs) and Webhooks manifests by running the commands `kubebuilder create api` and `kubebuilder create webhook` for the same group, kind and versions with the flag `--force`.   
+
+However, note that the tool will re-scaffold the files which means that you will lose its content. So, before executing the commands ensure that you have its content stored in another place. An  easy option is to use `git` to compare your locally changes with the previous versions in order to recover its contents.  
```

Comment:
The note warning of impeding file destruction should come before the commands that will do the destruction :) 

```suggestion
You can try to re-create the APIS(CRDs) and Webhooks manifests by using the `--force` flag.

Note, however, that the tool will re-scaffold the files which means that you will lose their content. Before executing the commands ensure that you have the files' content stored in another place. An easy option is to use `git` to compare your local change with the previous version to recover the contents.

Now, re-create the APIS(CRDs) and Webhooks manifests by running the  `kubebuilder create api` and `kubebuilder create webhook` for the same group, kind and versions with the flag `--force`, respectively.
```